install_config/install/advanced_install.adoc

```diff
@@ -776,15 +776,65 @@ openshift_examples_modify_imagestreams=true
 
 |`*openshift_docker_additional_registries*`
 |Specify the additional registry or registries.
+
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the URL and path to the directory where the image is located.
+
```

Comment:
I might say:
Specify the URL and path to namespace where the registry-console image is located.